sql/views.sql

```diff
@@ -55,12 +55,14 @@ CREATE OR REPLACE VIEW timescaledb_information.job_stats as
                    WHEN js.last_run_success = 'f' THEN 'Failed'
               END
     END as last_run_status,
-    CASE WHEN js.last_finish < '4714-11-24 00:00:00+00 BC' THEN 'Running'
+    CASE WHEN j.scheduled = false THEN 'Paused'
+         WHEN js.last_finish < '4714-11-24 00:00:00+00 BC' THEN 'Running'
          WHEN js.next_start IS NOT NULL THEN 'Scheduled'
```

Comment:
Shouldn't this be simply:

```suggestion
         WHEN j.scheduled = true THEN 'Scheduled'
```

otherwise, what is the meaning of `j.scheduled`?

Reply:
There is a change in semantics in 2.0 with the new "scheduled" field in bgw_job. Earlier the only possible states for a job are Scheduled Or Running. (If it  "Scheduled" state , then next_start field is set).
Now we have a third state.
So the view should reflect this new possibility :  ``` if j.scheduled = false then 'Not Scheduled' and then add the other options for the scheduled case ```

Reply:
Sven actually pointed out that scheduled may be false even if the job is already running, so I moved that to the second case.